Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:current
+
+RUN mkdir -p /app
+WORKDIR /app
+
+COPY package.json .
+RUN npm install -g npm@latest
```

Comment:
No need to install npm if it's already available.

Reply:
Strangely nom is not up to date in docker image, need to update it to be sure.